index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Block-Chainference</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="style_d.css">
+  <script src="main.js" defer></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+</head>
+
+<!--    Page for block chain coference 2023  -->
+
+<body>
+  <!--    bar menu and logo  -->
+  <header >
+    <nav class="nav-bar-grey">
+      <a href="" class="nav-bar-grey-fabook"></a>
+      <a href="" class="nav-bar-grey-twitter"></a>
+      <a href="" class="nav-bar-grey-language">Language</a>
+      <a href="" class="nav-bar-grey-mypage">My Page</a>
+      <a href="" class="nav-bar-grey-logout">Logout</a>
+    </nav>
+    <nav class="nav-bar">
+      <section class="nav-menuContainer">
+        
+        <button class="bugerBtn">
+          <div class="burger-icon line1"></div>
+          <div class="burger-icon line2"></div>
+          <div class="burger-icon line3"></div>
+        </button>
+        <!--burger menu-->
+        <ul class="burger-menu">
+          <div class="burger-menu-style">
+              <li class="burger-menu-exit-button"><button class="exitButton"></button></li>
```

Comment:
- [optional] wonderful job implementing the changes requested by the previous reviewer. as a bonus to make your work better, kindly add a margin left to the main headline section and align with your logo so it matches the original design.
- Also, please add the red line under the main program heading and add little margin right to the desktop nav items like it the design. nice work
yours
![Screenshot (1088)_LI](https://github.com/Zven94/conference-page/assets/65298048/86ace426-2b87-4afb-b05b-931a60dafe49)
original
![Screenshot (1090)](https://github.com/Zven94/conference-page/assets/65298048/7b2aa17a-5d9f-474e-b967-7acde4e950e1)

Reply:
Thank you for your time! I already addded the changes! 💪💪